Silicon/TigerlakePkg/Include/Register/Cpuid.h

```diff
@@ -0,0 +1,3952 @@
+/** @file

```

Comment:
Do we need whole this file?

Reply:
CPUID_VERSION_INFO
CPUID_INTEL_RDT_ALLOCATION
PUID_INTEL_RDT_ALLOCATION_L2_CACHE_SUB_LEAF
These 3 are used in Stage2BoardInitLib.c

Reply:
I removed all the unused parameters.